src/components/forms/FilterArchiveGroupsForm/FilterArchiveGroupsForm.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { reduxForm, Field } from 'redux-form';
+import { Alert, Well, Grid, Row, Col, Form } from 'react-bootstrap';
+
+import SubmitButton from '../SubmitButton';
+import { TextField, CheckboxField } from '../Fields';
+import { identity } from '../../../helpers/common';
+
+const FilterArchiveGroupsForm = ({
+  onSubmit = identity,
+  handleSubmit,
+  submitFailed = false,
+  submitSucceeded = false,
+  invalid
+}) =>
+  <Form method="POST" onSubmit={onSubmit}>
+    <Well bsSize="sm">
+      {submitFailed &&
+        <Alert bsStyle="danger">
+          <FormattedMessage
+            id="generic.operationFailed"
+            defaultMessage="Operation failed. Please try again later."
+          />
+        </Alert>}
+
+      <Grid fluid>
+        <Row>
+          <Col sm={12} md={4}>
+            <Field
+              name="search"
+              component={TextField}
+              label={
+                <span>
+                  <FormattedMessage
+                    id="app.filterArchiveGroupsForm.searchName"
+                    defaultMessage="Search by name"
+                  />:
+                </span>
+              }
+            />
+          </Col>
+          <Col sm={12} md={6} style={{ paddingTop: '28px' }}>
```

Comment:
There are some default classes in containers/App/recodex.css, including some generic margins and paddings. Check out, whether they won't be a better option.